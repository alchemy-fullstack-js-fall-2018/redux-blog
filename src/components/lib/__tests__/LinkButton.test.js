import React from 'react';
import { shallow } from 'enzyme';
import LinkButton from '../LinkButton.jsx';

describe('<LinkButton />', () => {
  test('renders', () => {
    const path = '/nowhere';
    const label = 'Go here';
    const wrapper = shallow(
      <LinkButton
        path={path}
        label={label}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
