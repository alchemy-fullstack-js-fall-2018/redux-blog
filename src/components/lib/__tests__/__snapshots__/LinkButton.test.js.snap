// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinkButton /> renders 1`] = `
<Styled(Link)
  to="/nowhere"
>
  Go here
</Styled(Link)>
`;
